src/views/_layouts/datahub-base.njk

```
@@ -41,7 +41,7 @@
       {% endif %}
       {% if feedbackLink %}
         <li class="proposition-menu__item">
-          <a href="{{ feedbackLink }}" class="proposition-menu__link{{ ' is-active' if '/support' in currentPath }}">Support</a>
+          <a href="{{ feedbackLink }}" class="proposition-menu__link{{ ' is-active' if currentPath and '/support' in currentPath }}">Support</a>
```

I don't think you need to check the variable is defined in Nunjucks. It fails silently.